src/_premake_init.lua

```diff
@@ -1483,6 +1476,18 @@
 		}
 	}
 
+	api.register {   -- DEPRECATED 2021-11-16
```

Comment:
Should it be `api.alias()` instead?

Reply:
I copied the pattern of `debuggerflavor` being deprecated in `vstudio/_preload.lua`. Let me know if I should do something different.